### ISSUE TITLE
Changing order of contact list by drag'n'drop / custom sorting

### ISSUE THREAD

**Issue by suhr** [qTox/qTox#645]
Just like in uTox.

**Comment by suhr** [comment]
Also I like moving contact list element as it is.

**Comment by zetok** [comment]
I think that it would require yet another way of "sorting view" beside `By Name` and `By Activity`, i.e. `Custom` ?

Or is there a better way to handle that?

**Comment by antis81** [comment]
This actually highly conflicts with sorting. Like @zetok says, a `custom` order would solve this, but do we really want that? @suhr Can you please explain the use case? To me it sounds, like you want to "favouritize" some contacts to appear on the top.

**Comment by suhr** [comment]
> To me it sounds, like you want to "favouritize" some contacts to appear on the top.

Yep, I want my own order of contacts or groupchats (especially groupchats).

**Comment by antis81** [comment]
I just sneaked into Pidgin/Adium, which also supports `custom` sorting. Never used it myself, but apparently it's a commonly used feature.
